<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="13" passed="12">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="43080" started-at="2017-02-21T15:06:43Z" finished-at="2017-02-21T15:07:26Z">
    <groups>
    </groups>
    <test name="Test1" duration-ms="42921" started-at="2017-02-21T15:06:43Z" finished-at="2017-02-21T15:07:26Z">
      <class name="com.ics.tests.referenceData.Sprint3_1">
        <test-method status="PASS" signature="test_74296()[pri:0, instance:com.ics.tests.referenceData.Sprint3_1@5649fd9b]" name="test_74296" duration-ms="3337" started-at="2017-02-21T15:06:43Z" finished-at="2017-02-21T15:06:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_74296 -->
        <test-method status="PASS" signature="test_74303()[pri:0, instance:com.ics.tests.referenceData.Sprint3_1@5649fd9b]" name="test_74303" duration-ms="221" started-at="2017-02-21T15:06:47Z" finished-at="2017-02-21T15:06:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_74303 -->
        <test-method status="PASS" signature="test_74304()[pri:0, instance:com.ics.tests.referenceData.Sprint3_1@5649fd9b]" name="test_74304" duration-ms="85" started-at="2017-02-21T15:06:47Z" finished-at="2017-02-21T15:06:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_74304 -->
        <test-method status="PASS" signature="test_74305()[pri:0, instance:com.ics.tests.referenceData.Sprint3_1@5649fd9b]" name="test_74305" duration-ms="19" started-at="2017-02-21T15:06:47Z" finished-at="2017-02-21T15:06:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_74305 -->
        <test-method status="PASS" signature="test_74306()[pri:0, instance:com.ics.tests.referenceData.Sprint3_1@5649fd9b]" name="test_74306" duration-ms="19" started-at="2017-02-21T15:06:47Z" finished-at="2017-02-21T15:06:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_74306 -->
        <test-method status="PASS" signature="test_74308()[pri:0, instance:com.ics.tests.referenceData.Sprint3_1@5649fd9b]" name="test_74308" duration-ms="17" started-at="2017-02-21T15:06:47Z" finished-at="2017-02-21T15:06:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_74308 -->
        <test-method status="PASS" signature="test_74309()[pri:0, instance:com.ics.tests.referenceData.Sprint3_1@5649fd9b]" name="test_74309" duration-ms="18" started-at="2017-02-21T15:06:47Z" finished-at="2017-02-21T15:06:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_74309 -->
        <test-method status="PASS" signature="test_74310()[pri:0, instance:com.ics.tests.referenceData.Sprint3_1@5649fd9b]" name="test_74310" duration-ms="17" started-at="2017-02-21T15:06:47Z" finished-at="2017-02-21T15:06:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_74310 -->
        <test-method status="PASS" signature="test_74319()[pri:0, instance:com.ics.tests.referenceData.Sprint3_1@5649fd9b]" name="test_74319" duration-ms="10881" started-at="2017-02-21T15:06:47Z" finished-at="2017-02-21T15:06:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_74319 -->
        <test-method status="PASS" signature="test_74320()[pri:0, instance:com.ics.tests.referenceData.Sprint3_1@5649fd9b]" name="test_74320" duration-ms="28257" started-at="2017-02-21T15:06:58Z" finished-at="2017-02-21T15:07:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_74320 -->
        <test-method status="FAIL" signature="test_74346()[pri:0, instance:com.ics.tests.referenceData.Sprint3_1@5649fd9b]" name="test_74346" duration-ms="27" started-at="2017-02-21T15:07:26Z" finished-at="2017-02-21T15:07:26Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Date doesn't exist or Status is not 0. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Test case failed. expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.ics.seleniumCoreUtilis.Component.assertTrue(Component.java:139)
	at com.ics.tests.referenceData.Sprint3_1.test_74346(Sprint3_1.java:303)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_74346 -->
      </class> <!-- com.ics.tests.referenceData.Sprint3_1 -->
    </test> <!-- Test1 -->
    <test name="Test2" duration-ms="51" started-at="2017-02-21T15:07:26Z" finished-at="2017-02-21T15:07:26Z">
      <class name="com.ics.tests.referenceData.Sprint3_2">
        <test-method status="PASS" signature="test_76191()[pri:0, instance:com.ics.tests.referenceData.Sprint3_2@5ec0a365]" name="test_76191" duration-ms="50" started-at="2017-02-21T15:07:26Z" finished-at="2017-02-21T15:07:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_76191 -->
      </class> <!-- com.ics.tests.referenceData.Sprint3_2 -->
    </test> <!-- Test2 -->
    <test name="Test3" duration-ms="81" started-at="2017-02-21T15:07:26Z" finished-at="2017-02-21T15:07:26Z">
      <class name="com.ics.tests.referenceData.Sprint3_3">
        <test-method status="PASS" signature="test_74379()[pri:0, instance:com.ics.tests.referenceData.Sprint3_3@3632be31]" name="test_74379" duration-ms="80" started-at="2017-02-21T15:07:26Z" finished-at="2017-02-21T15:07:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_74379 -->
      </class> <!-- com.ics.tests.referenceData.Sprint3_3 -->
    </test> <!-- Test3 -->
  </suite> <!-- Suite -->
</testng-results>
